CMakeLists.txt

```diff
@@ -776,6 +787,7 @@ add_subdirectory(src/util)
 
 list(APPEND GUI_SRC ${EXTRA_GUI_SRC})
 list(APPEND UTIL_SRC ${CMAKE_CURRENT_BINARY_DIR}/version.c)
+list(APPEND CORE_SRC ${CMAKE_CURRENT_BINARY_DIR}/flags.c)
```

Comment:
If `version.c` is in `UTIL_SRC` why is `flags.c` in `CORE_SRC`? (They should both be in `UTIL_SRC` since up to this point `CORE_SRC` is only for the emulation bits, not the periphery bits)